<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="186590d671cb.ant.amazon.com" ignored="0" name="Test" tests="3" failures="0" timestamp="30 Oct 2019 21:10:19 GMT" time="0.094" errors="0">
  <testcase name="@AfterTest printNoOfCustomersAvailedPromotion" time="0.003" classname="com.TestVagrant.onlineStore.StoreTests">
    <failure type="java.lang.AssertionError" message="Customer Availed promotion not equal expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: Customer Availed promotion not equal expected [3] but found [2]
at com.TestVagrant.onlineStore.StoreTests.printNoOfCustomersAvailedPromotion(StoreTests.java:100)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest printNoOfCustomersAvailedPromotion -->
  <testcase name="testScenario1" time="0.016" classname="com.TestVagrant.onlineStore.StoreTests"/>
  <testcase name="testScenario2" time="0.002" classname="com.TestVagrant.onlineStore.StoreTests"/>
  <testcase name="testScenario3" time="0.002" classname="com.TestVagrant.onlineStore.StoreTests"/>
</testsuite> <!-- Test -->
